.github/workflows/go.yml

```diff
@@ -32,5 +32,8 @@ jobs:
     - name: Build
       run: make clean all
 
+    - name: Setup
+      run: echo "ESTUARY_TOKEN=$(./estuary setup | grep Token | cut -d ' ' -f 3)" >> $GITHUB_ENV
+
     - name: Test
-      run: go test -v ./...
+      run: ESTUARY_TOKEN=$ESTUARY_TOKEN go test -v ./...
```

Comment:
hrm, `ESTUARY_TOKEN` should already be in the environment, i.e. adding `ESTUARY_TOKEN=$ESTUARY_TOKEN` before `go test` seems redundant.

if I'm reading the [docs](https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#example-6) correctly, you can probably just omit the this altogether

Reply:
Correct! Done and tested [here](https://github.com/gmelodie/estuary/actions/runs/1876540621)